src/main/java/com/capgemini/setrack/model/Booking.java

```diff
@@ -2,26 +2,35 @@
 
 import com.capgemini.setrack.model.enums.PaymentMethod;
 
-import java.time.LocalDate;
+import javax.persistence.*;
 import java.time.LocalDateTime;
+import java.util.List;
 
+@Entity
 public class Booking {
 
-    private long roomId;
-    private long guestId;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
     private LocalDateTime startDate;
     private LocalDateTime endDate;
     private LocalDateTime checkedIn;
     private LocalDateTime checkedOut;
     private LocalDateTime paid;
     private PaymentMethod paymentMethod;
 
+    @ManyToMany(mappedBy="bookings")
+    private List<Guest> guests;
+
+    @ManyToMany(mappedBy="bookings")
+    private List<Room> rooms;
 
     public Booking() {}
 
-    public Booking(long roomId, long guestId, LocalDateTime startDate, LocalDateTime endDate, LocalDateTime checkedIn, PaymentMethod paymentMethod) {
-        this.roomId = roomId; //roomId must be selected
-        this.guestId = guestId;//guestId must be selected
+    public Booking(List<Room> rooms, List<Guest> guests, LocalDateTime startDate, LocalDateTime endDate, LocalDateTime checkedIn, PaymentMethod paymentMethod) {
```

Comment:
Is een boeking met meerdere kamers echt voor alle kamers met dezelfde start/eindtijd en met ook dezelfde tijd voor het inchecken?